java/dataset/src/main/java/org/apache/arrow/dataset/jni/JniWrapper.java

```diff
@@ -67,7 +67,9 @@ private JniWrapper() {
   /**
    * Create Scanner from a Dataset and get the native pointer of the Dataset.
    * @param datasetId the native pointer of the arrow::dataset::Dataset instance.
-   * @param columns desired column names. Columns not in this list will not be emitted when performing scan operation.
+   * @param columns desired column names.
+   *                Columns not in this list will not be emitted when performing scan operation. Null equals
```

Comment:
Do you have any intention to insert a newline?

Reply:
Yes. This was to hard wrap at 100 columns I think.